layers/+lang/json/packages.el

```diff
@@ -32,6 +32,9 @@
   (use-package json-mode
     :defer t))
 
+(defun json/post-init-json-mode ()
```

Comment:
How this function is being called?

Reply:
Isn't it the same layer calling conventions magic that calls the `{layer}/pre-init-{pkg}` and `{layer}/init-{pkg}` packages?

Reply:
@tko you're right. Sorry, I was confused. Then why this `add-hook` call is not inside of `json/init-json-mode`?

Reply:
Actually this should be the other way around. This is the `post-init-prettier` that setup the hook. Indeed `json-init` does not know anything about `prettier` being used or not but `post-init-prettier` knows it implicitly as it is called only if `prettier` is used (i.e. the `prettier` layer is used).

Reply:
Yes, exactly 😸 Just like `web-beautify` is configured in the `json` layer.